src/app/(customerFacing)/layout.tsx

```diff
@@ -1,3 +1,6 @@
+"use client";
```

Comment:
Let's move "use client" into the new component you create.

The layout file is like a container for all the pages below it. If we keep it as "use client" it will make our entire app render on the client side